### PR TITLE
58 arcade UI css

### PR DESCRIPTION
**Pull request by OP-ReubenK**
this is to be merged up to 33

### CODE REVIEW

File: react-arcade/assets/img/neon_title.png
Comment (SamanthaCra) [inline react-arcade/assets/img/neon_title.png:1]: Did we confirm with Grayson that this is what he wants as the title?

File: react-arcade/src/App.jsx
Comment (SamanthaCra) [inline react-arcade/src/App.jsx:1]: Will this be adjusted to automatically pick up the number of games that can be played?

File: react-arcade/src/components/Game.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+// import React from "react";
```
Comment (fuzzyers) [inline react-arcade/src/components/Game.jsx:1]: Do we need this commented out code still here

File: react-arcade/src/components/App.scss [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+  background-size: contain;
+  margin: auto;
+  width: 50%;
+  height: 200px;;
+  // background-color: aqua;
+}
+
+// GameInfo styling
+
+#gameInfo {
```
Comment (fuzzyers) [inline react-arcade/src/components/App.scss:27]: Unused scss classes probably dont need to be here

File: react-arcade/src/components/App.scss
Comment (SamanthaCra) [inline react-arcade/src/components/App.scss:1]: Looks good, I understand that the commented out code is our alternative to the gif background.

**Review by fuzzyers — APPROVED** [review]
The application runs well and looks professional, try to keep the code tidy and well componentised with external functions to make the code more readable and add code comments.

**Review by SamanthaCra — APPROVED** [review]
All looks good. Well done!

**Pull request merged**